proof/access-control/ARM/ArchInterrupt_AC.thy

```diff
@@ -75,7 +74,7 @@ global_interpretation Interrupt_AC_1?: Interrupt_AC_1 "arch_authorised_irq_ctl_i
 proof goal_cases
   interpret Arch .
   case 1 show ?case
-    by (unfold_locales; (fact Interrupt_AC_assms)?)
+    by (unfold_locales; fact Interrupt_AC_assms)
```

Comment:
We should leave these in -- there are useful for debugging when the statement fails, because it will print a goal with the ones that could not be solved, whereas the non-? variant fails and only prints the original goal